projects/make-a-vis/src/lib/data-view/table/table.component.ts

```diff
@@ -37,6 +37,11 @@ export class TableComponent implements AfterViewInit, OnChanges, OnDestroy {
    */
   @Input() tableIndex: number;
 
+  /**
+   * Size of cell value (after which cell value will be truncated and replaced with an ellipsis)
+   */
+  @Input() cellValueSize = 64;
```

Comment:
This is not a good name. It doesn't clearly show that its a maximum or that the number is based on string length (is size pixels, inches, string length, etc??).

A more appropriate name would be `maxCellStringLength`